static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -124,15 +131,26 @@ function ConsoleMessage({
   );
 }
 
-const Common = styled('div')<{isLast: boolean; level: string}>`
+const Common = styled('div')<{
+  isLast: boolean;
+  level: string;
+  isEmitted?: boolean;
+}>`
   background-color: ${p =>
     ['warning', 'error'].includes(p.level)
       ? p.theme.alert[p.level].backgroundLight
       : 'inherit'};
-  color: ${p =>
-    ['warning', 'error'].includes(p.level)
-      ? p.theme.alert[p.level].iconHoverColor
-      : 'inherit'};
+  color: ${({isEmitted = true, ...p}) => {
+    if (!isEmitted) {
+      return p.theme.gray200;
+    }
```

Comment:
What about instead of changing text color, it was opacity?

We should also add a css `transition`.

Reply:
Oh, I decided to go with color as it is calculated at build time (better performance because browser doesn't have to calculate the color on the fly).